war/pom.xml

```diff
@@ -353,8 +353,14 @@ THE SOFTWARE.
                 </artifactItem>
                 <artifactItem>
                   <groupId>org.jenkins-ci.plugins</groupId>
+                  <artifactId>display-url-api</artifactId>
+                  <version>1.0</version>
```

Comment:
Why? Users do not care; the changes in 2.0 were done for architectural reasons. The plugin versions here are the minimum required to pick up all security advisories, plus the minimum transitive dependencies. Which in this case is 1.0. Certainly newer versions of plugins have all sorts of improvements and bug fixes that users may care about, and they are free to update to them at their leisure.